app/assets/javascripts/controllers/live_metrics/util/metrics-parse-url-factroy.js

```diff
@@ -4,10 +4,12 @@ angular.module('miq.util').factory('metricsParseUrlFactory', function() {
     var pathname = $window.location.pathname.replace(/\/$/, '');
     dash.providerId = '/' + (/^\/[^\/]+\/([r\d]+)$/.exec(pathname)[1]);
 
-    // TODO: get this values from GET/POST values ?
+    // parse window's url
+    dash.params = (new URL($window.location)).searchParams;
+
     dash.tenantList = [];
-    dash.minBucketDurationInSecondes = 20 * 60;
-    dash.max_metrics = 10000;
-    dash.items_per_page = 10;
+    dash.minBucketDurationInSecondes = parseInt(dash.params.get("bucket")) || (20 * 60);
```

Comment:
`parseInt` really needs `, 10`, so as not to try parsing octal and hexadecimal notations.